Tasks/DownloadFileShareArtifactsV0/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,39 @@
+{
+  "loc.friendlyName": "Download Build Artifacts",
+  "loc.helpMarkDown": "",
+  "loc.description": "Download Build Artifacts",
+  "loc.instanceNameFormat": "Download Build Artifacts",
+  "loc.group.displayName.advanced": "Erweitert",
+  "loc.input.label.buildType": "Download artifacts produced by",
+  "loc.input.help.buildType": "Download artifacts produced by the current build, or from a specific build.",
+  "loc.input.label.project": "Project",
+  "loc.input.help.project": "The project from which to download the build artifacts",
+  "loc.input.label.definition": "Build definition",
+  "loc.input.help.definition": "Select the build definition name",
+  "loc.input.label.specificBuildWithTriggering": "When appropriate, download artifacts from the triggering build.",
+  "loc.input.help.specificBuildWithTriggering": "If checked, this build task will try to download artifacts from the triggering build. If there is no triggering build from the specified definition, it will download artifacts from the build specified in the options below.",
+  "loc.input.label.buildId": "Build",
```

Comment:
strings in other languages don't match strings in EN-US. for e.g. this one isn't there in EN-US. can you check?

Reply:
updated